src/OpenSage.Game/Graphics/Animation/AnimationInstance.cs

```diff
@@ -111,11 +120,11 @@ private void UpdateBoneTransforms(in TimeInterval gameTime)
             {
                 if (Reverse)
                 {
-                    time -= gameTime.DeltaTime;
+                    time -= gameTime.DeltaTime * _speedFactor;
```

Comment:
Please could you put `gameTime.DeltaTime * _speedFactor` into a variable, to avoid duplicating the multiplication here and in the `else` branch?

Reply:
done